src/devices/sunricher.ts

```diff
@@ -55,6 +67,32 @@ async function syncTime(endpoint: Zh.Endpoint) {
 }
 
 const definitions: Definition[] = [
+    {
+        zigbeeModel: ['HK-SENSOR-4IN1-A'],
+        model: 'HK-SENSOR-4IN1-A',
+        vendor: 'Sunricher',
+        description: '4IN1 Sensor',
+        fromZigbee: [fz.ias_occupancy_alarm_1, fz.battery, fz.temperature, fz.humidity, fz.illuminance],
+        toZigbee: [],
+        extend: [
+            deviceEndpoints({'endpoints': {'1': 1, '2': 2, '3': 3, '4': 4, '5': 5}}),
+            battery(),
+            identify(),
+            occupancy(),
+            iasZoneAlarm({'zoneType': 'generic', 'zoneAttributes': ['alarm_1', 'alarm_2', 'tamper', 'battery_low']}),
+            temperature({'endpointNames': ['3']}),
+            humidity({'endpointNames': ['4']}),
+            illuminance({'endpointNames': ['5']})],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint3 = device.getEndpoint(3);
+            const endpoint4 = device.getEndpoint(4);
+            const endpoint5 = device.getEndpoint(5);
+            await endpoint3.bind('msTemperatureMeasurement', coordinatorEndpoint);
+            await endpoint4.bind('msRelativeHumidity', coordinatorEndpoint);
+            await endpoint5.bind('msIlluminanceMeasurement', coordinatorEndpoint);
```

Comment:
This shouldn't be needed, `illuminance()` takes care of this. You can remove the whole configure